cmd/experimental/kjobctl/apis/v1alpha1/application_profile_types.go

```diff
@@ -93,7 +93,8 @@ type TemplateReference string
 // +kubebuilder:validation:XValidation:rule="!has(self.requiredFlags) || !('ntasks' in self.requiredFlags) || self.name == 'Slurm'", message="ntasks flag can be used only on Slurm mode"
 // +kubebuilder:validation:XValidation:rule="!has(self.requiredFlags) || !('output' in self.requiredFlags) || self.name == 'Slurm'", message="output flag can be used only on Slurm mode"
 // +kubebuilder:validation:XValidation:rule="!has(self.requiredFlags) || !('partition' in self.requiredFlags) || self.name == 'Slurm'", message="partition flag can be used only on Slurm mode"
-// +kubebuilder:validation:XValidation:rule="!has(self.requiredFlags) || !('priority' in self.requiredFlags) || self.name == 'Slurm'", message="priority flag can be used only on Slurm mode"
+// +kubebuilder:validation:XValidation:rule="!has(self.requiredFlags) || !('priority' in self.requiredFlags) || self.name != 'Interactive'", message="priority flag can't be used only on Interactive mode"
+// +kubebuilder:validation:XValidation:rule="!has(self.requiredFlags) || !('skip-priority-validation' in self.requiredFlags) || self.name != 'Interactive'", message="skip priority validation flag can't be used only on Interactive mode"
```

Comment:
```suggestion
```

`--skip-priority-validation` is not required flag. So we don't need to add here.